app/main.py

```diff
@@ -1,3 +1,87 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+
+from app.customer import Customer
+from app.shop import Shop
+from app.car import Car
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as file:
+        shopping_data = json.load(file)
+
+    fuel_cost = shopping_data["FUEL_PRICE"]
+    customer_list = []
+    shop_list = []
+
+    for customer in shopping_data["customers"]:
+        person = Customer(customer["name"],
+                          customer["product_cart"],
+                          customer["location"],
+                          customer["money"],
+                          Car(customer["car"]["brand"],
+                              customer["car"]["fuel_consumption"],
+                              fuel_cost)
+                          )
+        customer_list.append(person)
```

Comment:
Use list comprehension and unpacking (using `**`)
Hint: You can place Car creation inside Customer init